test/unit/services/services_spec.js

```diff
@@ -0,0 +1,121 @@
+"use strict";
+
+var Instance = require("../../../lib/services/services-db");
+var _        = require("lodash");
+var expect   = require("chai").expect;
+var Genesis  = require("genesis");
+var Bluebird = require("bluebird");
+var Sinon    = require("sinon");
+require("sinon-as-promised")(Bluebird);
+
+describe("the Adapter class ", function () {
+	it("is frozen", function () {
+		expect(Object.isFrozen(Instance), "frozen").to.be.true;
+	});
+
+	function describeCreate (description, options, fails) {
+		var optionsToPass = JSON.parse(JSON.stringify(options));
+		var expectedOptions = _.defaults(
+			options,
+			{
+				id    : undefined,
+				type  : "t2.micro",
+				ami   : "ami-d05e75b8",
+				state : "pending",
+				uri   : null
+			}
+		);
+
+		describe("is creating a new instance " + description, function () {
```

Comment:
Syntactically, "is" is not necessary in this description (the printed test run will say "The Adapter class is creating a new instance returns an instance", rather than "The Adapter class creating a new instance returns an instance", which is a bit more natural, although admittedly still not going to get past an English professor).